tests/admin_inlines/admin.py

```diff
@@ -292,6 +293,21 @@ class TeacherAdmin(admin.ModelAdmin):
     inlines = [StudentInline]
 
 
+# model for #9107
+class InlineOneForHolder6Admin(admin.TabularInline):
+    model = InlineOneForHolder6
+
+
+class InlineTwoForHolder6Admin(admin.StackedInline):
+    model = InlineTwoForHolder6
+
+
+class Holder6Admin(admin.ModelAdmin):
+    inlines = [InlineOneForHolder6Admin, InlineTwoForHolder6Admin]
+
+
+# register for #9107
+site.register(Holder6, Holder6Admin)
```

Comment:
Are you able to reuse existing models? The names also don't seem very descriptive. What is "Holder6"?

Reply:
@pope1ni  I tried but I wanted to have a many to many field in the inline formset to test this feature. That is why I have created this model.